tests/unit/testipaddress.cpp

```diff
@@ -172,6 +226,35 @@ void TestIpAddress::excludeAddresses_data() {
       << "10.0.0.0/8"
       << "0.0.0.0/5,11.0.0.0/8,12.0.0.0/6,128.0.0.0/1,16.0.0.0/4,32.0.0.0/"
          "3,64.0.0.0/2,8.0.0.0/7";
+
+  QTest::addRow("world vs ::1")
```

Comment:
This is a bit of a nightmare to make sense of. It's hard to see why this would be correct. Can it be re-ordered in such a way to show the pattern (eg: increasing prefix length).

Reply:
in theory... it's alphabetic order :) But we can sort it by prefix length.